### ISSUE TITLE
go tests do not compile

### ISSUE THREAD

**Issue by maljub01** [benlaurie/objecthash#28]
Running:
```
GOPATH=`pwd` go get golang.org/x/text/unicode/norm
GOPATH=`pwd` go test -v \
                                 go/objecthash/objecthash.go \
                                 go/objecthash/objecthash_test.go
```

Results in:
```
# command-line-arguments
go/objecthash/objecthash.go:32:20: objectHash(e) used as value
go/objecthash/objecthash.go:49:18: objectHash(o) used as value
go/objecthash/objecthash.go:77:26: objectHash(k) used as value
go/objecthash/objecthash.go:78:26: objectHash(v) used as value
go/objecthash/objecthash_test.go:50:19: objectHash redeclared in this block
        previous declaration at go/objecthash/objecthash.go:148:44
FAIL    command-line-arguments [build failed]
make: *** [go] Error 2
```

This is because `go/objecthash/objecthash_test.go` redefines the `objecthash` method and changes its signature from `func ObjectHash(obj interface{}) [hashLength]byte` to `func objectHash(o interface{})`.

This causes the error where the code in `objecthash.go` which uses its return value does not compile anymore.

**Comment by maljub01** [comment]
This can be fixed by renaming the method `objectHash` in `objecthash_test.go`.

**Comment by maljub01** [comment]
This was fixed in c8469f886112de9543bfa8e464df1539f36536dc
